src/css/profile/mobile/common/progress.less

```diff
@@ -14,6 +14,8 @@ tau-progress {
 
 	margin-left: 32 * @unit_base;
 	margin-right: 32 * @unit_base;
+	margin-top: (32 - 3) / 2 * @px_base;
```

Comment:
I think that instead of use `margin` you should set widget height. Height of progress bar is defined in guideline as `32`